web/styles/portico/portico_signin.css

```diff
@@ -668,7 +668,8 @@ html {
     }
 }
 
-#new-user-email-address-visibility {
+#new-user-email-address-visibility,
+#full_name_help_text {
```

Comment:
We could probably define a common class like `registration-form-hint` for controlling this CSS in a more clean/semantic fashion than just putting it on two selectors.